includes/admin/class-wcmp-admin.php

```diff
@@ -34,6 +34,8 @@ class WCMP_Admin
     public const META_HS_CODE                = "_myparcel_hs_code";
     public const META_COUNTRY_OF_ORIGIN      = "_myparcel_country_of_origin";
 
+    public const ORDER_STATUS_DELIVERED      = "afgeleverd";
```

Comment:
in plaats van `afgeleverd ` kan je beter een ID gebruiken. Als de klant een Engelse versie heeft geïnstalleerd dan zal dit vertaald worden naar Delivered. Waardoor jouw code niet zal gaan werken. Probeer in plaats van een tekst een status ID te gebruiken.